scss/main_screen/_navbar_wrapper.scss

```diff
@@ -83,7 +83,7 @@
     font-size: 18px;
   }
 
-  #main-menu-button {
+  #three-dot-menu-button {
```

Comment:
can we leave it as three-dot menu? Because it's not really a main menu ^^

Reply:
?